### ISSUE TITLE
constantly getting "java runtime could not be located" messages

### ISSUE THREAD

**Issue by mttindustries** [Umoxfo/vscode-w3cvalidation#19]
I've been constantly getting this constantly getting "java runtime could not be located" messages evertime I load VSC. I've followed all the instructions in the plugin details page. I've installed the java development platform and i've pointed all my settings to the correct location 
C:\Program Files\Java\jdk-13.0.2

but it's still not working

**Comment by Umoxfo** [comment]
Sorry, I could not reproduce your issue.

Please reinstall the latest JDK and add ";C:\Program Files\Java\jdk-13.0.2\bin" to the environment variable PATH.

**Comment by mttindustries** [comment]
yeah I tried that. Reinstalled JDK, updated the environmental variable path to include the \bin directory on the end. Tried updating viscode settings to do the same. didn't work. Tried reinstalling the extension, didn't work. tried it without the /bin/ in settings... still doesn't work

**Comment by Umoxfo** [comment]
Please try to restart the VSCode in order to load reliably the changed environment variables.

**Comment by mttindustries** [comment]
yes i've done that too each step. I even installed and new installation of viscode, restarted the computer even and it still does it

**Comment by FSamolev** [comment]
Have the same problem...

**Comment by colvacdir** [comment]
I'm also getting the error, but only on an earlier version of VSC. The latest insider build doesn't show the error? They are both using the EXACT same path.

W3C Validation: 2.6.1
macOS: 10.12.6
Java: 1.8.0_241-b07
Path: /Library/Java/JavaVirtualMachines/jdk1.8.0_241.jdk/Contents/Home/jre

Visual Studio Code: 1.41.1
Error: "Java runtime could not be located."

Visual Studio Code: 1.44.0-insider
No Error

**Comment by Umoxfo** [comment]
Sorry for the inconvenience.
It lacked testing in the macOS environment, so I set up GitHub Actions to test in a cross-platform environment.
